packages/wrangler/src/deployment-bundle/apply-middleware.ts

```diff
@@ -68,7 +68,7 @@ export async function applyMiddlewareLoaderFacade(
 				export * from "${prepareFilePath(entry.file)}";
 
 				export const __INTERNAL_WRANGLER_MIDDLEWARE__ = [
-					...(OTHER_EXPORTS.__INJECT_FOR_TESTING_WRANGLER_MIDDLEWARE__ ?? []),
+					${process.env.NODE_ENV === "test" ? `...(OTHER_EXPORTS.__INJECT_FOR_TESTING_WRANGLER_MIDDLEWARE__ ?? []),` : ""}
```

Comment:
Does Vitest set this?

Reply:
Yes it does!

https://github.com/vitest-dev/vitest/blob/e2d69c63b290841fff741a30b0025405e0d4c68f/docs/guide/migration.md?plain=1#L282

Reply:
yup